tensorflow_serving/model_servers/main.cc

```diff
@@ -200,7 +200,8 @@ void RunServer(int port, std::unique_ptr<ServerCore> core,
   std::shared_ptr<grpc::ServerCredentials> creds = InsecureServerCredentials();
   builder.AddListeningPort(server_address, creds);
   builder.RegisterService(&service);
-  builder.SetMaxMessageSize(tensorflow::kint32max);
+  // Raise max frame size for larger request if it is reasonable
+  builder.SetMaxMessageSize(104857600);
```

Comment:
static const int32 kint32max = ((int32)0x7FFFFFFF);
It's 4M.
2GB is the ProtoBuf's hard limit.